packages/cli/index.js

```diff
@@ -183,21 +183,24 @@ async function prepareMinimalAppEnv (appName, dbName, redisName, port, sport, se
     await updateAppEnvFile(appName, 'COOKIE_SECRET', `${appName}-secret`)
 }
 
-async function prepareAppEnvLocalAdminUsers (appName) {
+async function prepareAppEnvLocalAdminUsers (appName, identity = 'email') {
     // TODO(pahaz): check create-user.js logic!
-    const justUserEmail = await getAppEnvValue(appName, 'DEFAULT_TEST_USER_IDENTITY') || 'user@example.com'
+    const justUserDefaultIdentity = identity === 'email' ? 'user@example.com' : '+79067777777'
+    const justUserIdentity = await getAppEnvValue(appName, 'DEFAULT_TEST_USER_IDENTITY') || justUserDefaultIdentity
     const justUserPassword = await getAppEnvValue(appName, 'DEFAULT_TEST_USER_SECRET') || getRandomString()
     const justUserOpts = JSON.stringify({ password: justUserPassword, isAdmin: false, name: 'JustUser' })
-    await safeExec(`yarn workspace @app/${appName} node ./bin/create-user.js ${JSON.stringify(justUserEmail)} ${JSON.stringify(justUserOpts)}`)
-    const adminUserEmail = await getAppEnvValue(appName, 'DEFAULT_TEST_ADMIN_IDENTITY') || 'admin@example.com'
+    await safeExec(`yarn workspace @app/${appName} node ./bin/create-user.js ${JSON.stringify(justUserIdentity)} ${JSON.stringify(justUserOpts)}`)
+
+    const adminDefaultIdentity = identity === 'email' ? 'admin@example.com' : '+79068888888'
```

Comment:
it's better to raise an error if we can't find any required data

Reply:
It's not required. In case, where you're launching `prepare.js` for a first time, there'll be no identities in `.env`.
So the app will use default values (from initialData in old good times) and then update `.env`